src/zenml/zen_stores/schemas/step_run_schemas.py

```diff
@@ -227,6 +232,7 @@ def to_model(
             input_artifacts=input_artifacts,
             output_artifacts=output_artifacts,
             metadata=metadata,
+            step_logs=self.step_logs.to_model(),
```

Comment:
```suggestion
            step_logs=self.step_logs.to_model() if self.step_logs,
```

step logs is optional so I can imagine that self.step_logs might be none and it would fail here